config/deploy/prod.rb

```diff
@@ -1,7 +1,9 @@
-# frozen_string_literal: true
+ # frozen_string_literal: true
```

Comment:
remove space

Reply:
Whoops, new keyboard is taking some getting used to... my bad. Fixed.